MRN-Code/coinstac
# moved consortia results call from dashboard-home to redux action

* **Problem:**
   1. Dashboard status-items are loaded slowly after initial load despite the fact that they should be quickly retrievable from local storage.
   1. Dashboard displays *No computation results* message and no spinner despite the fact that there are computation results still loading.
* **Solution:**
   1. Changed remote results request to pull from redux action and add to props rather than pulling on each load and adding results to state.
* **Tests:**
   1. Login to application and note the status-items loading on the right-side panel.
   1. Navigate into the *Consortia* tab, then back into the *Home* tab. Results should load immediately if model still exists.
   1. Navigate into the any of the status-items' results, then navigate back to *Home*. If model reloads, you should see the spinner rotating until remote results have been retrieved.
